metrics/web/docs/features/storybook/01_metrics_storybook_design.md

```diff
@@ -0,0 +1,220 @@
+
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Feasibility study](#feasibility-study)
+    - [Requirements](#requirements)
+    - [Landscape](#landscape)
+        - [Use existing packages](#use-existing-packages)
+            - [Storyboard](#storyboard)
+            - [Monarch](#monarch)
+            - [Dashbook](#dashbook)
+            - [Storybook_flutter](#storybook_flutter)
+        - [Manually create a Storybook from scratch](#manually-create-a-storybook-from-scratch)
+        - [Decision](#decision)
+    - [Prototyping](#prototyping)
+    - [System modeling](#system-modeling)
+
+# Analysis
+> Describe a general analysis approach.
+
+This analysis purposes to discover the feasibility of implementing the feature and the requirements list for the feature to implement. The Metrics Storybook feature requires defining the set of UI elements of the Metrics Web Application that we should present on the storybook. 
+
+The analysis examines the possible solutions for the feature implementation and compares these solutions using the project's architecture rules and principles. Then, the document states the general approach of implementing the Metrics Storybook and defines whether it should be a custom implementation or the existing solution.
+
+### Feasibility study
+> A preliminary study of the feasibility of implementing this feature.
+
+Flutter allows us to create separate widgets that we can reuse and combine later to build a specific project. Also, we can build an interface to display these widgets to end-users and control their appearance through themes and properties.
+
+Given the above, it's possible to create the `Metrics storybook`.
+
+This feature will be useful to the following end-users: developers, designers, and clients.
+
+First of all, the great purpose of the Metrics storybook is that each widget and story (a specific state of a widget) can be easily reused across the entire project by any developer who works on it, ensuring consistent design and UX. Metrics storybook creating allows developers to work in parallel on widgets and do not touch the main Metrics Web Application's codebase. It also, makes it easier to add new widgets, since we can test them separately and only then make changes to the code of the application.
+
+The storybook application can be shared with and commented on by designers so that they can add their input regarding the implementation of the designs.
+
+Another group that can use this feature is clients. Storybook makes it easy to show pieces of the software to keep the client in the loop. Even small pieces of UI can be shared and made available for feedback, preventing longer periods without any deliverables.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+Before defining requirements to the feature, we should determine which components from the Metrics Web Application should be included in the storybook.
```

Comment:
:tw:
```suggestion
Before defining requirements to the feature, we should determine which components from the Metrics Web Application the storybook should include.
```
Please, prefer active voice to passive voice. See [Google Technical Writing](https://developers.google.com/tech-writing/one/active-voice#prefer_active_voice_to_passive_voice) for more details.